### ISSUE TITLE
Dragging path is not working in Chrome 55 with latest version of core Leaflet (1.0.3)

### ISSUE THREAD

**Issue by peppierre** [w8r/Leaflet.Path.Drag#29]
## How to reproduce

Leaflet version I'm using: 1.0.3
Browser (with version) I'm using: Chrome55
OS/Platform (with version) I'm using: Android
Leaflet.Path.Transform version: 1.0.2
Leaflet.Path.Drag version: 1.0.6

## What behaviour I'm expecting and which behaviour I'm seeing

Note that both Transform and Drag plugin (in this order) is referred.
Prior to Chrome 55 and using core Leaflet v1.0.2, plugin's _onDragStart() handler is called when a path is started to be dragged, as it is expected. In Chrome55, dragging paths doesn't work due to Chrome's newly introduced pointer events, I guess.
Unfortunately, updating to v1.0.3 of core did not fix this behavior in Chrome 55, dragging of path still doesn't work because event handler above is simply not called anymore.

**Comment by IvanSanchez** [comment]
If this needs any changes upstream, please leave comments at https://github.com/Leaflet/Leaflet/issues/5298

**Comment by w8r** [comment]
Seems to be Android-specific? Cause I cannot reproduce it.

**Comment by IvanSanchez** [comment]
Yeah, I can reproduce it on Android. Seems that using a mouse pointer will make the browser fire both `mousedown` and `pointerdown` events, but using a touchscreen will fire only `pointerdown`s, no `touchdown`s.

**Comment by Damorck** [comment]
Same issue here. So this is possible to fix? And is it working on Android with Leaflet 0.7?
